### DISCUSSION TITLE
release pakage not found

### DISCUSSION THREAD

**Discussion by peng19832** [community/community#69901] [Packages]
### Select Topic Area

Bug

### Body

![image](https://github.com/community/community/assets/9590209/5a6e6781-6be8-4528-af88-76ffddb11e62)
release not found?

**Comment by thevindu-w** [comment]
Did you mean [https://github.com/hhyo/Archery/archive/refs/tags/v1.10.0.tar.gz](https://github.com/hhyo/Archery/archive/refs/tags/v1.10.0.tar.gz) ?
Or the release [https://github.com/hhyo/Archery/releases/tag/v1.10.0](https://github.com/hhyo/Archery/releases/tag/v1.10.0) ?

**Reply by peng19832** [reply]
first one.
can't download the tar.

**Reply by thevindu-w** [reply]
I tried it and it downloaded without any issue (downloaded ~3.9MB tar.gz file). The link [https://github.com/hhyo/Archery/archive/refs/tags/v1.10.0.tar.gz](https://github.com/hhyo/Archery/archive/refs/tags/v1.10.0.tar.gz) works for me.
But the link shown in the question [https://github.com/hhyo/Archery/tar.gz/refs/tags/v1.10.0](https://github.com/hhyo/Archery/tar.gz/refs/tags/v1.10.0) gives not found.
